<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMedia</string>
  </property>
  <widget class="QWidget" name="QWidget_centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="QTabWidget_tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="Photo" name="QWidget_Tab_Photo">
       <attribute name="title">
        <string>Photo</string>
       </attribute>
      </widget>
      <widget class="MediaPlayer" name="QWidget_Tab_MediaPlayer">
       <attribute name="title">
        <string>Media Player</string>
       </attribute>
      </widget>
      <widget class="MediaRecorder" name="QWidget_Tab_MediaRecorder">
       <attribute name="title">
        <string>Media Recorder</string>
       </attribute>
      </widget>
      <widget class="Credits" name="QWidget_Tab_Credits">
       <attribute name="title">
        <string>Credits</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="QStatusBar_statusBar"/>
  <widget class="QMenuBar" name="QMenuBar_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="QMenu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="QAction_OpenImage"/>
    <addaction name="QAction_OpenMedia"/>
    <addaction name="separator"/>
    <addaction name="QAction_Exit"/>
   </widget>
   <widget class="QMenu" name="QMenu_View">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="QActionPhotoTab"/>
    <addaction name="QActionMediaPlayerTab"/>
    <addaction name="QAction_MediaRecorderTab"/>
    <addaction name="QAction_CreditsTab"/>
   </widget>
   <addaction name="QMenu_File"/>
   <addaction name="QMenu_View"/>
  </widget>
  <action name="QAction_OpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="QAction_OpenMedia">
   <property name="text">
    <string>Open Media</string>
   </property>
  </action>
  <action name="QAction_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="QActionPhotoTab">
   <property name="text">
    <string>Photo tab</string>
   </property>
  </action>
  <action name="QActionMediaPlayerTab">
   <property name="text">
    <string>Media player tab</string>
   </property>
  </action>
  <action name="QAction_MediaRecorderTab">
   <property name="text">
    <string>Media Recorder tab</string>
   </property>
  </action>
  <action name="QAction_CreditsTab">
   <property name="text">
    <string>Credits tab</string>
   </property>
  </action>
  <action name="QAction_Credits">
   <property name="text">
    <string>Credits</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Photo</class>
   <extends>QWidget</extends>
   <header>Photo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MediaPlayer</class>
   <extends>QWidget</extends>
   <header>MediaPlayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MediaRecorder</class>
   <extends>QWidget</extends>
   <header>MediaRecorder.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Credits</class>
   <extends>QWidget</extends>
   <header>Credits.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
